### PR TITLE
Update readme to reflect move from incubating/temporary to 'in production use' for this repository

### PR DESCRIPTION
**Pull request by QuintinWillison**

### CODE REVIEW

**Review by m-hulbert — APPROVED** [review]
LGTM!

**Review by tbedford — APPROVED** [review]
Only a couple of minor comments/observations.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
-# Canonical table
+# Ably's Canonical SDK API Reference Commentaries
```
[the remark this answers is not in the document]
Comment (Srushtika) [reply README.md:1]: @tbedford just FYI - we are addressing this naming discrepancy in the IA hill :)

**Review by Srushtika — APPROVED** [review]
LGTM, thanks @QuintinWillison!

**Review by owenpearson — APPROVED** [review]

**Pull request merged**